Project.toml

```diff
@@ -16,7 +16,7 @@ StatsModels = "3eaba693-59b7-5ba5-a881-562e759f1c8d"
 [compat]
 DataFrames = "0.19, 0.20, 0.21, 0.22, 1"
 Distributions = "0.23, 0.24, 0.25"
-GLM = "1.3"
+GLM = "1.3, 1.8"
```

Comment:
This shouldn't be necessary. "1.3" describes all version in [v1.3.0 - 2.0.0). The only time you need to list out versions if for pre-1.0 releases like Distributions.jl above.

Output from running `test` locally to confirm:

```
(Bootstrap) pkg> test Bootstrap
     Testing Bootstrap
      Status `/private/var/folders/sy/srj5df5j7jvf0_hb225496sh0000gn/T/jl_rOQUHl/Project.toml`
  [e28b5b4c] Bootstrap v2.3.3 `~/repos/invenia/Bootstrap.jl`
  [a93c6f00] DataFrames v1.5.0
  [31c24e10] Distributions v0.25.86
  [38e38edf] GLM v1.8.1
  [ce6b1742] RDatasets v0.7.7
  [2913bbd2] StatsBase v0.33.21
  [3eaba693] StatsModels v0.6.33
  [9a3f8284] Random `@stdlib/Random`
  [10745b16] Statistics `@stdlib/Statistics`
  [8dfed614] Test `@stdlib/Test`
      Status `/private/var/folders/sy/srj5df5j7jvf0_hb225496sh0000gn/T/jl_rOQUHl/Manifest.toml`
  [e28b5b4c] Bootstrap v2.3.3 `~/repos/invenia/Bootstrap.jl`
  [336ed68f] CSV v0.10.9
  [49dc2e85] Calculus v0.5.1
  [324d7699] CategoricalArrays v0.10.7
  [d360d2e6] ChainRulesCore v1.15.7
  [9e997f8a] ChangesOfVariables v0.1.6
  [944b1d66] CodecZlib v0.7.1
  [34da2185] Compat v4.6.1
  [a8cc5b0e] Crayons v4.1.1
  [9a962f9c] DataAPI v1.14.0
  [a93c6f00] DataFrames v1.5.0
  [864edb3b] DataStructures v0.18.13
  [e2d170a0] DataValueInterfaces v1.0.0
  [b429d917] DensityInterface v0.4.0
  [31c24e10] Distributions v0.25.86
  [ffbed154] DocStringExtensions v0.9.3
  [fa6b7ba4] DualNumbers v0.6.8
  [e2ba6199] ExprTools v0.1.9
  [5789e2e9] FileIO v1.16.0
  [48062228] FilePathsBase v0.9.20
  [1a297f60] FillArrays v0.13.7
  [59287772] Formatting v0.4.2
  [38e38edf] GLM v1.8.1
...
```

Reply:
Quite right - not sure why I missed that!